puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/rpm_packaging.groovy

```diff
@@ -24,7 +24,13 @@ pipeline {
             steps {
 
                 script {
-                    changed_packages = sh(returnStdout: true, script: "git diff origin/${ghprbTargetBranch} --name-only -- 'packages/**.spec' | xargs dirname | xargs -n1 basename |sort -u").trim()
+                    changed_packages = sh(returnStdout: true, script: "git diff origin/${ghprbTargetBranch} --name-only -- 'packages/**.spec'")
+
+                    if (changed_packages) {
+                        changed_packages = sh(returnStdout: true, script: "${changed_packages} | xargs dirname | xargs -n1 basename |sort -u").trim()
```

Comment:
should that be `echo ${changed_packages}`?

Reply:
it should https://ci.theforeman.org/blue/organizations/jenkins/foreman-packaging-rpm-pr-test/detail/foreman-packaging-rpm-pr-test/22/pipeline/